grafana/target.js

```diff
@@ -204,4 +204,9 @@ Target.prototype.summarize15min = function summarize15min() {
     return this.summarize('15min');
 };
 
+Target.prototype.hide = function hide() {
+    this.hide = true;
```

Comment:
Actually, this won't work. When the panels are serialized, the `hide` property needs to be set there, not on the target.